qiskit_ibm_runtime/qiskit/primitives/__init__.py

```diff
@@ -0,0 +1,54 @@
+# This code is part of Qiskit.
```

Comment:
This is copied from terra. Don't review this file. Will be removed from here when terra code is released.